dist/polkit-1/rules.d/org.freedesktop.policykit.pkexec.run-zincati-deadend.policy

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policyconfig PUBLIC
+ "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/PolicyKit/1/policyconfig.dtd">
+<policyconfig>
+  <action id="org.freedesktop.policykit.pkexec.run-zincati-deadend">
+    <description>Allow Zincati to write dead-end release information</description>
+    <defaults>
+      <allow_any>yes</allow_any>
+      <allow_inactive>yes</allow_inactive>
+      <allow_active>yes</allow_active>
+    </defaults>
```

Comment:
I think this should ideally be removed or at least set to no as in this current form this allows anyone to call the action.

Reply:
From https://wiki.archlinux.org/index.php/Polkit#Actions

Reply:
Yeah, makes sense. I was under the impression that this action will get called only for the `zincati` user when invoked by the `zincati.rules`(https://github.com/coreos/zincati/pull/367/files#diff-fb15afa4bf5dbf6d827a99f2d5d1af3eR13)

Reply:
WDYT about this change?
```
                        <allow_any>no</allow_any>
                        <allow_inactive>no</allow_inactive>
                        <allow_active>auth_admin_keep</allow_active>
```

Reply:
I think it should always be no for all entries here as this will never be called interactively (zincati is a daemon) and we only want to allow the case covered in the additional polkit rule we add.